package.json

```
@@ -1,28 +1,28 @@
 {
-    "name": "next-netlify-platform-starter",
-    "version": "0.1.0",
-    "private": true,
-    "scripts": {
-        "dev": "next dev",
-        "build": "next build",
-        "start": "next start",
-        "lint": "next lint"
-    },
-    "dependencies": {
-        "@netlify/blobs": "^8.2.0",
-        "blobshape": "^1.0.0",
-        "bright": "^0.8.5",
-        "markdown-to-jsx": "^7.4.5",
-        "next": "15.5.4",
-        "react": "18.3.1",
-        "react-dom": "18.3.1",
-        "unique-names-generator": "^4.7.1"
-    },
-    "devDependencies": {
-        "@tailwindcss/postcss": "^4.0.15",
-        "eslint": "8.57.1",
-        "eslint-config-next": "15.5.4",
-        "postcss": "^8.4.36",
-        "tailwindcss": "^4.0.15"
-    }
+  "name": "next-netlify-platform-starter",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev --turbopack",
```

turbopack is now default for both dev and build!
```suggestion
    "dev": "next dev",
```